### PR TITLE
fix: if xml value is set, change value in class metadata

### PR DESCRIPTION
**Pull request by gfuentesboost**

### CODE REVIEW

**Comment by gfuentesboost** [comment]
what else is required for the PR?

**Review by franmomu — APPROVED** [review]
Looks good

File: tests/Gedmo/Mapping/Xml/ReferencesMappingTest.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
+use Doctrine\ORM\Mapping\Driver\XmlDriver;
+use Doctrine\Persistence\Mapping\Driver\MappingDriverChain;
+use Gedmo\References\ReferencesListener;
+use Gedmo\Tests\Mapping\Fixture\Xml\References;
+use Gedmo\Tests\Tool\BaseTestCaseOM;
+
+/**
+ * These are mapping tests for SoftDeleteable extension
```
[the remark this answers is not in the document]
Comment (gfuentesboost) [reply tests/Gedmo/Mapping/Xml/ReferencesMappingTest.php]: Sorry, I didn't notice that comment, fixed.

**Review by phansys — APPROVED** [review]

**Comment by phansys** [comment]
Thank you @gfuentesboost!

**Pull request merged**